rocket/templates/rocket/activation.html

```diff
@@ -0,0 +1,14 @@
+{% extends "base.html" %}
```

Comment:
This file addition looks good. Please send this as an independent patch/commit so that I can accept it.

Reply:
Sorry, I couldn't understand your intention from independent patch/commit. Are you want a new PR for this?